target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -335,6 +335,11 @@ tl-wr841n-v13)
 	ucidef_set_led_switch "lan4" "lan4" "$boardname:green:lan4" "switch0" "0x10"
 	ucidef_set_led_switch "wan" "wan" "$boardname:green:wan" "switch0" "0x01"
 	;;
+tplink,tl-wr841n-v14)
+	ucidef_set_led_wlan "wlan2g" "wlan2g" "$boardname:green:wlan" "phy0tpt"
```

Comment:
Having LED triggers defined in DTS is preferred, so please use `linux,default-trigger = "phy0tpt";` in the DTS instead, for example:

https://github.com/openwrt/openwrt/blob/e65f935de9e67ea07a251f114455f12e9555c5bf/target/linux/ath79/dts/ar9344_comfast_cf-e120a-v3.dts#L36-L40

Reply:
How do I set the port_mask for a switch trigger in the DTS?

Reply:
I'm talking here about the wlan LED only.

Reply:
ah ok. strangely it does not work with the trigger set by the DTS file.

```
root@OpenWrt:/sys/devices/platform/leds/leds/tl-wr841n-v14:green:wlan# cat trigger 
[none] switch0 timer default-on netdev phy0rx phy0tx phy0assoc phy0radio phy0tpt 
root@OpenWrt:/sys/devices/platform/leds/leds/tl-wr841n-v14:green:wlan# 
```